packages/sc-toolkit/commands/init/src/deploy.ts

```diff
@@ -0,0 +1,34 @@
+import * as dotenv from 'dotenv';
+import { readFileSync } from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { deploySC, WalletClient, ISCData } from '@massalabs/massa-sc-deployer';
+import { Args } from '@massalabs/massa-web3';
+
+dotenv.config();
+
+if (!process.env.WALLET_PRIVATE_KEY || !process.env.JSON_RPC_URL_PUBLIC) {
+  throw new Error('Missing WALLET_PRIVATE_KEY in .env file');
+}
+
+const deployerAccount = await WalletClient.getAccountFromSecretKey(
+  process.env.WALLET_PRIVATE_KEY,
+);
+
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
why a __ prefix ?

Reply:
To match the naming : https://www.geeksforgeeks.org/node-js-__filename-object/